cocotb/share/makefiles/Makefile.inc

```diff
@@ -75,10 +81,17 @@ ifeq ($(SIM_DEFINE),XCELIUM)
 SIM_DEFINE = IUS
 endif
 
-export LIB_DIR=$(COCOTB_PY_DIR)/cocotb/libs
```

Comment:
Or phrased another way, did this line need to move?

Reply:
Legacy